### ISSUE TITLE
Undefined array key in ParsedownFilter

### ISSUE THREAD

**Issue by serial** [Zavy86/WikiDocs#69]
Hey Zavi,
I noticed a disproportionate number of entries in the webserver error log with the following message:

`[proxy_fcgi:error] AH01071: Got error 'PHP message: PHP Warning:  Undefined array key "name" in [abspath]/libraries/parsedown-filter-0.0.1/ParsedownFilter.php on line 23`

I did not checked what the file is exactly for, but can you implement something like `isset` or `is_array` there to check for array key exists.

It is throwing multiple times on page loads, so the error log is just getting bigger.

**Comment by Zavy86** [comment]
my fault.. can you try version 1.0.5 please?

**Comment by serial** [comment]
Some are still coming from [here](https://github.com/Zavy86/WikiDocs/blob/bb0bfa19a4fb707db96786a6a034641ebf54be5a/libraries/parsedown-filter-0.0.1/ParsedownFilter.php#L23-L28)

If I modify and surround them with the following
```php
if (isset($Element['name'])) {
  if ( is_string( $Element['name'] ) ) { 
  ...
}
```
all is fine and no errors are coming anymore.

**Comment by Zavy86** [comment]
nice.. fixed completely in 1.0.6
